test/unit/proxy_api/bmc_test.rb

```diff
@@ -217,4 +217,40 @@ def setup
     @testbmc.expects(:get).with(expected_path, data).at_least_once
     @testbmc.lan_mac(@options)
   end
+
+  test "#power should return true when result was true" do
+    action = 'cycle'
+    data = @options.merge({:action => action})
+    @testbmc.stubs(:put).returns(fake_rest_client_response(
+      {
+        'action' => 'cycle',
+        'result' => true
+      }
+    ))
+    assert_equal true, @testbmc.power(data)
+  end
+
+  test "#power should return true when result was ok" do
+    action = 'cycle'
```

Comment:
This would be more DRY (and easier to read IMO) if you could extract the first part of this to a method, something like `stub_bmc_response(result)` and call that everywhere. 

The fact all tests start with '#power' suggests you probably want to group them in a `context '#power'` block